bb.edn

```diff
@@ -121,6 +121,7 @@
                 ["JARS=~/my-stuff ./bin/mage jar-download 1.45.2" "Download metabase_1.45.2.jar to ~/my-stuff"]]
    :requires   [[mage.jar-download :as jar-download]]
    :options    [["-r" "--run" "run the new jar after downloading it"]
+                ["-e" "--env FILE" "you can optionally provide a lein-env file to specify the environment to use when running the jar."]
```

Comment:
not the biggest fan of the name "env" since it's so overloaded. what do you think of `--lein-env` and `-l`? I won't gatekeep on it tho

Reply:
I'll go with `--env-file` since if/when we want to support a different env file format, I think it should be under the same argument and it detect the correct file type.